### PR TITLE
Setting permissions to the NFS mount paths

### PR DESCRIPTION
**Pull request by Anand-Reddy7**

### CODE REVIEW

**Comment by digvijay-ukirde** [comment]
@rajan-mis With the integration of LDAP user, we are expecting some extra permission so that LDAP user can use the NFS mount to do read write operation. From a security perspective we are reducing the NFS option from no_root_sqush to root_squash. Let me know if you have any concerns.

File: roles/nfs_file_share/tasks/nfs_file_share.yml [hunk 1]
```diff
@@ -20,13 +20,20 @@
...
     sleep 10              #This is temporary fix for Dbus error.
   with_items: "{{ export_list_exists.results }}"
   register: created_nfs_export
   when: is_admin_node | default(false) == true and item.rc != 0
   run_once: true
 
+- name: Update permissions for directories in export list
+  shell: |
+    find {{ scale_protocols.mountpoint }}/{{ item.item }} -type d ! -path "{{ scale_protocols.mountpoint }}/{{ item.item }}/.snapshots" -exec chmod 777 {} +
+  with_items: "{{ export_list_exists.results }}"
```
Comment (rajan-mis) [inline roles/nfs_file_share/tasks/nfs_file_share.yml:33]: @Anand-Reddy7 do we have any common validation where we are checking` scale_protocols.mountpoint`  is defined or not.

Comment (Anand-Reddy7) [reply roles/nfs_file_share/tasks/nfs_file_share.yml:33]: Yes @rajan-mis we have validation in the main.tf file. Please take a look here : https://github.com/Anand-Reddy7/ibm-spectrum-scale-install-infra/blob/anand_mount_permission/roles/nfs_file_share/tasks/main.yml#L5

FYI @digvijay-ukirde

Comment (Anand-Reddy7) [reply roles/nfs_file_share/tasks/nfs_file_share.yml:33]: @rajan-mis defined "scale_protocols.mountpoint is defined" on when condition.

**Review by rajan-mis — APPROVED** [review]

**Comment by Anand-Reddy7** [comment]
We are going to raise one more PR with all the changes. So, I am closing it. 

FYI @digvijay-ukirde

**Pull request closed without merging**